jobs/keepalived/templates/config/keepalived.conf.erb

```diff
@@ -9,7 +9,7 @@ vrrp_script chk_sfnode {
 vrrp_instance VI_1 {
     debug 2
     interface eth0 # interface to monitor
-    <% if ( spec.bootstrap === TRUE ) %>
+    <% if ( spec.ip == p('master_ip') ) %>
```

Comment:
put `virtual_router_id 51 # Assign one ID for this route` in common place.
Also following block can be changed 

```
    unicast_peer {
    <% if ( spec.ip == p('slave_ip') ) %>
        <%= p('master_ip')  %> # peer IP
    <% else %>
        <%= p('slave_ip')  %> # peer IP
    <% end %>
    }
```
to have only slave ip in case of spec.ip is master and vice-versa